lib/core/terraform_config/gvc.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module TerraformConfig
+  class Gvc < Base
+    attr_reader :name, :description, :tags, :domain, :locations, :pull_secrets, :env, :load_balancer
+
+    # rubocop:disable Metrics/ParameterLists
```

Comment:
Nit: This can be moved to the end of line 17. That way we don't have to enable the rule again later, it gets disabled only for the block.